nrfs/include/internal/services/nrfs_gdpwr.h

```diff
@@ -15,9 +15,11 @@ extern "C" {
 
 /** @brief Power domain selection */
 typedef enum __NRFS_PACKED {
-	GDPWR_POWER_DOMAIN_ACTIVE_FAST,
-	GDPWR_POWER_DOMAIN_ACTIVE_SLOW,
-	GDPWR_POWER_DOMAIN_MAIN_SLOW
+	GDPWR_GD_FAST_ACTIVE_0,
```

Comment:
I think these should be numbered like here to allow backwards compatibility if there is changes in the future. This is now leading to a breaking change which in theory could have been avoided
https://github.com/zephyrproject-rtos/hal_nordic/pull/263/files#diff-c22a6f2ed9dee39b4c06992d6ca2e695d842b0558ec163660a5168293a06d10eR17